src/repositories/helpers/MessageBuilder.ts

```diff
@@ -0,0 +1,60 @@
+import { MessageActionRow, MessageButton, MessageEmbed } from "discord.js";
+import { BallChaser } from "../../types/common";
+
+export default class MessageBuilder {
+  
+  // eslint-disable-next-line max-len
+  static readonly normIconURL = "https://raw.githubusercontent.com/mattwells19/UNCC-Six-Mans.js/main/media/norm_still.png";
+
+  static readonly queueButtons = new MessageActionRow()
+    .addComponents(
+      new MessageButton()
+        .setCustomId("joinQueue")
+        .setLabel("Join")
+        .setStyle("SUCCESS"),
+      new MessageButton()
+        .setCustomId("leaveQueue")
+        .setLabel("Leave")
+        .setStyle("DANGER"),
+    );
+
+  static leaderboardMessage(leaderboardInfo : string[]) : MessageEmbed[] {
+
+    const embeds = leaderboardInfo.map((content, index) => {
+      const embedCtr = leaderboardInfo.length > 1 ? `(${index + 1}/${leaderboardInfo.length})` : "";
+  
+      return new MessageEmbed()
+        .setColor("BLUE")
+        .setTitle(`UNCC 6 Mans | Leaderboard ${embedCtr}`.trim())
+        .setThumbnail(this.normIconURL)
+        .setDescription("```" + content + "```");
+    });
+
+    return embeds;
+  }
+
+  static activeQueueMessage( ballchasers : ReadonlyArray<Readonly<BallChaser>> ) : MessageEmbed {
+    
+    const ballChaserNames = ballchasers.map(function (a) { return a.name; }).join("\n");
```

Comment:
Can we make this an arrow function and use a better variable name than `a`, please? Just makes it a bit easier to read.
`ballChasers.map((ballChaser) => ballChaser.name).join("\n");`

Reply:
Fixed in next commit.